build_tools/kokoro/gcp_ubuntu/cmake-bazel/linux/x86-swiftshader/build.sh

```diff
@@ -61,7 +61,9 @@ echo "Configuring CMake"
    -DIREE_BUILD_SAMPLES=OFF \
    -DIREE_BUILD_XLA_COMPILER=ON \
    -DIREE_BUILD_TFLITE_COMPILER=ON \
-   -DIREE_BUILD_TENSORFLOW_COMPILER=ON .
+   -DIREE_BUILD_TENSORFLOW_COMPILER=ON \
+   -DIREE_ENABLE_OLD_INTEGRATION_TESTS=ON \
+   -DIREE_ENABLE_TFLITE_INTEGRATION_TESTS=ON .
```

Comment:
nit: move the `.` to a new line so it doesn't show up in diffs when the last option is changed?

```suggestion
   -DIREE_BUILD_TENSORFLOW_COMPILER=ON \
   -DIREE_ENABLE_OLD_INTEGRATION_TESTS=ON \
   -DIREE_ENABLE_TFLITE_INTEGRATION_TESTS=ON \
   .
```